packages/graphql-js-operation-payload-generator/src/__tests__/RelayMockPayloadGenerator.test.ts

```diff
@@ -921,11 +934,12 @@ describe("with @relay_test_operation", () => {
     );
   });
 
-  xtest("generate mock with Mock Resolvers for Interface Type with multiple fragment spreads", () => {
+  test("generate mock with Mock Resolvers for Interface Type with multiple fragment spreads", () => {
```

Comment:
This test best illustrates the change. Previously the type condition of the first fragment that was found was used, in this case `User`. Now the `Page` fragment is selected instead.